### DISCUSSION TITLE
Cannot get PTZ Control to talk to the camera

### DISCUSSION THREAD

**Discussion by Chipperchap** [sircharlo/meeting-media-manager#803] [Q&A] [answered]
Hi - I am wondering if someone clever can help me please. 

I have set up several PTZ cameras in the past and have usually figured out what is going on to get them to work with OBS / PTZ Control dock.  But this instance is driving me crazy!  Please see [video here](https://capture.dropbox.com/jQlOQO00Dec0pgDE) and if anyone has any tips of what I am doing wrong - something obvious that I am missing...? 

Kind regards
Simon

**Comment by mtdvlpr** [comment, marked as answer]
Hmm, couple (maybe obvious) things you could try:
- Are you running the latest version of ptz-controls plugin?
- Have you tried to use the VISCA port (seeing as you're configuring a VISCA TCP connection)
- Have you tried other connection types? We use the connection over UDP.

I'm just spitballing here.

The difference between UDP and TCP is that TCP checks if the whole message has come through and either fails completely or succeeds, making it slower, but more reliable. Usually, for camera setups, you use UDP which just transfers the message without checking if everything went through correctly

**Reply by Chipperchap** [reply]
I don't ever remember trying the "VISCA port" - found it in the configuration settings. Tried the number and.... SUCCESS!  
Yay! Thank you so much @mtdvlpr !  Soo grateful! My frustration is over - what a relief! 😅

I can now look at installing the camera in our city sister K-Hall while they wait for a new one to arrive. 😊

**Reply by mtdvlpr** [reply]
Glad I could help :)
